lib/matplotlib/ticker.py

```diff
@@ -168,6 +169,15 @@
     long = int
 
 
+# Work around numpy/numpy#6127.
+def divmod(x, y):
```

Comment:
Based on your argument, we would never use underscores.  There are two reasons for using one here:
1) With its present name, this function is shadowing a builtin.  
2) It is a workaround for a bug.  Some time after the bug is fixed, we will want to remove this function.  If it has an underscore, we don't need to go through a deprecation process, or argue about whether to do so.

Also, I see that divmod is used in 3 other modules; is there reason to use this version in those locations as well?  If so, then it could be put in cbook.